src/runtime_src/core/edge/user/zynq_dev.h

```diff
@@ -16,6 +16,8 @@
 #ifndef _XCL_ZYNQ_DEV_H_
 #define _XCL_ZYNQ_DEV_H_
 
+#include <boost/filesystem.hpp>
+#include <regex>
 #include <string>
 #include <vector>
 #include <fstream>
```

Comment:
I am not sure fstream is used in the header either.

Reply:
A member of class zynq_device uses it